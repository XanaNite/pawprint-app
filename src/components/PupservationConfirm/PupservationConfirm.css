* {
    box-sizing: border-box;
}

.pupservationConfirm-container {
    text-align: center;
    border-top: 1px solid lightgray;
    margin: 40px;
    padding: 20px;
    background: #214A77;
}

.pupservationConfirm-link{
    text-decoration: none;
    font-weight: bolder;
    color: white;    
}

