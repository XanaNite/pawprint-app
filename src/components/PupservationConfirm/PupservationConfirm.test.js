import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import PupservationConfirm from './PupservationConfirm';

describe(`PupservationConfirm component`, () => {
  const props = {
    className: 'test-class-name',
    children: <p>test children</p>,
    'data-other': 'test-other-prop'
  };

  it('renders PupservationConfirm by default', () => {
    const wrapper = shallow(<PupservationConfirm />)
    expect(toJson(wrapper)).toMatchSnapshot()
  });

  it('renders the PupservationConfirm given props', () => {
    const wrapper = shallow(<PupservationConfirm {...props} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  });
})
