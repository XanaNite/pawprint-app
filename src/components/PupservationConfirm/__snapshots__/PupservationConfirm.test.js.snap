// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PupservationConfirm component renders PupservationConfirm by default 1`] = `
<div
  className="pupservationConfirm-container"
>
  <Link
    className="pupservationConfirm-link"
    to="/profile/1"
  >
    Submit
  </Link>
</div>
`;

exports[`PupservationConfirm component renders the PupservationConfirm given props 1`] = `
<div
  className="pupservationConfirm-container"
>
  <Link
    className="pupservationConfirm-link"
    to="/profile/1"
  >
    Submit
  </Link>
</div>
`;
